### PR TITLE
patatt: new, 0.6.3

### PR DESCRIPTION
**Pull request by xtexx**
Topic Description
-----------------

- patatt: bump REL for topic Revision Marking Guidelines
- patatt: new, 0.6.3

Package(s) Affected
-------------------

- patatt: 0.6.3-1

Security Update?
----------------

No

Build Order
-----------

```
#buildit patatt
```

Test Build(s) Done
------------------

**Primary Architectures**

- [x] Architecture-independent `noarch`

### CODE REVIEW

File: app-devel/patatt/autobuild/defines [hunk 1]
```diff
@@ -0,0 +1,7 @@
+PKGNAME=patatt
+PKGSEC=devel
+PKGDEP="git setuptools pynacl"
+PKGDES="Cryptographic patch attestation utilities for the masses"
```
Comment (MingcongBai) [inline app-devel/patatt/autobuild/defines]: Drop "for the masses."

**Review by MingcongBai — CHANGES_REQUESTED** [review]

**Comment by xtexx** [comment]
LGTM on AMD64
image

**Review by MingcongBai — APPROVED** [review]

File: app-devel/patatt/autobuild/defines [hunk 2]
```diff
@@ -0,0 +1,7 @@
+PKGNAME=patatt
+PKGSEC=devel
+PKGDEP="git setuptools pynacl"
```
Comment (MingcongBai) [inline app-devel/patatt/autobuild/defines]: Shouldn't `git` and `setuptools` be build-time dependencies?

**Review by MingcongBai — CHANGES_REQUESTED** [review]

File: app-devel/patatt/autobuild/defines (hunk 2 above)
Comment (xtexx) [reply app-devel/patatt/autobuild/defines]: Moved setuptools to build-time dependency. Git is kept.

**Review by MingcongBai — APPROVED** [review]

**Pull request merged**